src/pages/Home.jsx

```diff
@@ -1,6 +1,136 @@
+                <h2 className="text-4xl font-semibold text-center mb-10">WHAT DOES FUSION DO?</h2>
+                <div className="fusion-row">
+                    <div className="fusion-text-box">
+                        <h3 className="text-xl font-semibold mb-2">Insert Title Here</h3>
+                        <p className="mb-4">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+                        <button className="fusion-button">BUTTON</button>
+                    </div>
+                    <div className="fusion-image-box">
+                        <img src={placeholderDarkImage} alt="Placeholder" className="fusion-image" />
+                    </div>
+                </div>
+                <div className="fusion-row">
+                    <div className="fusion-image-box">
+                        <img src={placeholderDarkImage} alt="Placeholder" className="fusion-image" />
+                    </div>
+                    <div className="fusion-text-box">
+                        <h3 className="text-xl font-semibold mb-2">Insert Title Here</h3>
+                        <p className="mb-4">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+                        <button className="fusion-button">BUTTON</button>
+                    </div>
+                </div>
+                <div className="fusion-row">
+                    <div className="fusion-text-box">
+                        <h3 className="text-xl font-semibold mb-2">Insert Title Here</h3>
+                        <p className="mb-4">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+                        <button className="fusion-button">BUTTON</button>
+                    </div>
+                    <div className="fusion-image-box">
+                        <img src={placeholderDarkImage} alt="Placeholder" className="fusion-image" />
+                    </div>
+                </div>
+            </div>
+            {/* Join Fusion */}
+            <div className="relative w-full h-screen flex flex-col justify-center items-center text-center bg-cover bg-center" style={{ backgroundImage: `url(${placeholderDarkImage})` }}>
+                <div className="relative z-10 flex flex-col items-center justify-center h-full text-black">
+                    <h2 className="text-4xl font-bold">JOIN FUSION</h2>
+                    <p className="text-2xl mt-4">WE ARE ALWAYS MORE OPEN TO PEOPLE JOINING US.</p>
```

Comment:
in the build, the join fusion tag is quite small even though its specified for the text, might want to review your css module and try experimenting with this